mobile_verifier/src/reward_shares.rs

```diff
@@ -770,6 +772,36 @@ pub fn get_scheduled_tokens_for_oracles(duration: Duration) -> Decimal {
     get_total_scheduled_tokens(duration) * ORACLES_PERCENT
 }
 
+fn eligible_for_coverage_map(
+    oracle_boosting_status: OracleBoostingStatus,
+    speedtests: &[Speedtest],
+    trust_scores: &[LocationTrust],
+) -> bool {
+    if oracle_boosting_status == OracleBoostingStatus::Banned {
+        return false;
+    }
+
+    let avg_speedtest = Speedtest::avg(speedtests);
+    if avg_speedtest.tier() == SpeedtestTier::Fail {
+        return false;
+    }
+
+    let avg_trust_score = average_trust_score(trust_scores);
```

Comment:
I think you want to use `coverage_point_calculator::location::multiplier();`, you'll need to make it `pub`. 

It handles CBRS always being trusted regardless of having any LocationTrust. But it may be a good idea to handle empty lists for non-CBRS radios.

Reply:
Fixed